daemon/src/engine/qcommon/q_shared.cpp

```diff
@@ -1347,35 +1347,12 @@ Safe strncpy that ensures a trailing zero
 =============
 */
 
-#ifndef NDEBUG
```

Comment:
What's wrong with this code? I don't think this is specific to Hunk Debug, but rather to any Q_strncpyz across the engine.